rllib/evaluation/tests/test_rollout_worker.py

```diff
@@ -1037,11 +1051,11 @@ def test_wrap_multi_agent_env(self):
     def test_no_training(self):
         class NoTrainingEnv(MockEnv):
             def __init__(self, episode_length, training_enabled):
-                super(NoTrainingEnv, self).__init__(episode_length)
+                super().__init__(episode_length)
                 self.training_enabled = training_enabled
 
             def step(self, action):
-                obs, rew, done, info = super(NoTrainingEnv, self).step(action)
```

Comment:
This syntax was enabled with Python 3.0. I wonder why it's not part of linters for modern Python projects.

Reply:
yeah